### ISSUE TITLE
Uncaught exceptions from CustomSpokenForms.updateSpokenFormMaps?

### ISSUE THREAD

**Issue by bjaspan** [cursorless-dev/cursorless#2261]
I am working on some changes to VscodeFileSystem.ts. When running the tests, I'm seeing `rejected promise not handled within 1 second: NeedsInitialTalonUpdateError: Custom spoken forms file not found at /var/folders/g5/2jjc2rf56rg29d286ttd05_000fy9v/T/20ae72e7d885e78e1d632cfd8c725972/state.json. Using default spoken forms.` in the logs. I thought this was due to my changes (since they are called by the code that raises this error), though after debugging for a while I just discovered that the problem happens on the main branch as well.  I believe this is because CustomSpokenForms.updateSpokenFormMaps catches errors from this.talonSpokenForms.getSpokenFormEntries() but then re-throws them, and the callers of CustomSpokenForms.updateSpokenFormMaps do not catch them.  Is this a (known?) bug?

**Comment by pokey** [comment]
Does this cause a test failure or just log spam during running tests?

**Comment by bjaspan** [comment]
Main gets 7082 passing tests, so I guess just log spam.

**Comment by pokey** [comment]
yeah my guess is that's testing a failure mode, and we're not properly suppressing the error message. We can leave this open as it's better to suppress that, but prob not urgent

**Comment by saidelike** [comment]
Interestingly I don't see this error anymore when running the tests after this commit b9709f9ad3bf23cd431e3d1549a72a5428d78fc2

**Comment by saidelike** [comment]
Nevermind I do see them, but it's a lot less frequent.
